src/components/transformer-template/TransformerTemplate.tsx

```diff
@@ -88,6 +91,8 @@ export type TransformerTemplateInit = {
   textInput2?: TextInputInit;
   dropdown1?: DropdownInit;
   dropdown2?: DropdownInit;
+  expressionPrompt1?: ExpressionPromptInit;
+  expressionPrompt2?: ExpressionPromptInit;
```

Comment:
Since the way these are implemented they're just generic text that's inserted into the component and don't have to specifically be next to an expression, what if we just called them `prompt1` and `prompt2` or `text1` and `text2`?